src/cosalib/kubevirt.py

```diff
@@ -12,7 +34,9 @@ def kubevirt_run_ore_replicate(*args, **kwargs):
 
 
 def kubevirt_cli(parser):
-    """
-    Extend a parser with the KubeVirt options
-    """
+    parser.add_argument("--name",
+                        help="Name to append to the repository (e.g. fedora-coreos). Defaults to the build name.")
+    parser.add_argument("--repository", help="repository to push to (e.g. quay.io or quay.io/myorg)", required=True)
+    parser.add_argument('--no-dry-run', dest='dry_run', action='store_false')
+    parser.set_defaults(dry_run=True)
```

Comment:
To clarify on this: I think having a dry run knob is good, but for consistency with the other commands I don't I think it should be the default. So let's just have a single opt-in `--dry-run` knob?